RELEASE.md

```diff
@@ -1,39 +1,51 @@
 # Releasing the Jaeger Operator for Kubernetes
 
-1. Update Jaeger version in `versions.txt`
+Steps to release a new version of the Jaeger Operator:
 
-1. Make sure the new version is present at `pkg/upgrade/versions.go`
 
-1. Prepare a changelog since last release. Get the `OAUTH_TOKEN` from (Github)[https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token] and select `repo:status` scope.
+1. Change the `versions.txt `so that it lists the target version of the Jaeger (if it is required). Don't touch the operator version it will be changed automatrically in the next ste.
 
-    ```
-    OAUTH_TOKEN=... make changelog
-    ```
+2. Run `make prepare-release VERSION=1.25.0`, using the operator version that will be released.
+
+3. Prepare a changelog since last release. 
 
-1. Commit version change and changelog and create a pull request:
+4. Commit the changes and create a pull request:
 
    ```
    git commit -sm "Preparing release v1.25.0"
    ```
 
-1. Tag and push
+5. Once the changes above are merged and available in `master` tag it with the desired version, prefixed with `v`, eg. `v1.25.0`
 
     ```
-    git checkout master ## it's only possible to release from master for now!
-    git tag release/v1.25.0
-    git push git@github.com:jaegertracing/jaeger-operator.git release/v1.25.0
+    git checkout master
+    git tag v1.25.0
+    git push git@github.com:jaegertracing/jaeger-operator.git v1.25.0
     ```
 
-1. Wait until release CI job finishes and then pull the changes:
+6. The GitHub Workflow will take it from here, creating a GitHub release and publishing the images
 
     ```
     git pull git@github.com:jaegertracing/jaeger-operator.git master
     ```
 
-1. Apply generated OLM catalog files to [operatorhub.io](https://operatorhub.io)
+7. After the release, PRs needs to be created against the Operator Hub Community Operators repositories:
+
+    * One for the `upstream-community-operators`, used by OLM on Kubernetes. (https://github.com/k8s-operatorhub/community-operators)
+    * One for the `community-operators` used by OpenShift. (https://github.com/redhat-openshift-ecosystem/community-operators-prod)
+
+This can be done with the following steps:
+- Update master `git pull git@github.com:jaegertracing/jaeger-operator.git master`
+- Clone both repositories `upstream-community-operators` and `community-operators` 
```

Comment:
add link to these repositories

Reply:
The links are in the above paragraph, Do you think we need to add it here too?